deltachat-jsonrpc/src/api.rs

```diff
@@ -231,6 +231,20 @@ impl CommandApi {
         Ok(())
     }
 
+    /// Performs a background fetch for all accounts in parallel with a timeout.
+    ///
+    /// The `AccountsBackgroundFetchDone` event is emitted at the end,
+    /// process all events until you get this one and you can safely return to the background
+    /// without forgeting to create notifications caused by timing race conditions.
+    async fn background_fetch_for_all_accounts(&self, timeout_in_seconds: f64) -> Result<()> {
```

Comment:
better use the same, consistent method names as in ffi:

```suggestion
    async fn accounts_background_fetch(&self, timeout_in_seconds: f64) -> Result<()> {
```

(i see that the the "for_all_accounts" suffix is already in use, however, it is a little cumbersome wording and maybe not worth to be set in stone further)